File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2020w/ID07IO2AB/Strategy/src/Sportsmen.java
public class Sportsmen
{
   private final String Name;
   private final SportsmenStrategyRun ssRun;
   public Sportsmen(String Name,SportsmenStrategyRun ssRun)
   {
      this.Name=Name;
      this.ssRun=ssRun;
   }
   String run() { return ssRun.doit(); }
   String jump() { return "Jumping"; }
   String swim() { return "Swimming"; }
   @Override public String toString() { return Name; }
}

File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06IO2/Command/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    class Sportsmen
    {
        //public string _name;
        //public string name { get { return _name; } private set { _name = value; } }
        public string name { get; private set; }
        public Sportsmen(string name)
        {
            this.name = name;
        }
        public string Run() { return "Runing"; }
        public string Swim() { return "Swimming"; }
        public string Jump() { return "Jumping"; }
        public string execute(SportsmenCommand command)
        {
            return name + ": " + command.execute(this);
        }
    }
}
